### PR TITLE
Feature/time aesthetics and endtime

### PR DESCRIPTION
**Pull request by arastogi15**
**- Adds:**

-  sub-10s decimal timing
- end state text at end of turn
- a fun lil favicon

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -23,7 +23,7 @@ button {
 #container {
   width: 90%;
   margin: auto;
-  max-width: 300px;
+  /* max-width: 300px; */
```
Comment (bstanfield) [inline index.css:26]: go ahead and delete this line

File: index.js [hunk 2]
```diff
@@ -1,31 +1,50 @@
...
+const PAUSE_TEXT = "Pause";
+const PLAY_TEXT = "Resume play";
+
+// needed to account for button press time
+const START_BUFFER = 0.1;
+let secondsLeft = TURN_DURATION + START_BUFFER;
 let paused = false;
 let countdown;
 
+// TODO: add decimal point when it's 10 seconds or less
```
Comment (bstanfield) [inline index.js]: remove

File: index.css (hunk 1 above)
Comment (arastogi15) [reply index.css:26]: ok bro

**Review by arastogi15 — COMMENTED** [review]
Thank bro

**Pull request merged**